src/Compilers/Test/Resources/Core/CompilerTestResources.csproj

```diff
@@ -346,6 +346,7 @@
     <EmbeddedResource Include="SymbolsTests\snKey2.snk" />
     <EmbeddedResource Include="SymbolsTests\snPublicKey.snk" />
     <EmbeddedResource Include="SymbolsTests\snPublicKey2.snk" />
```

Comment:
Two files are getting checked in but only one is showing up in the project file.